### DISCUSSION TITLE
How to add custom values for memory and ephemeral storage for custodian lambdas

### DISCUSSION THREAD

**Discussion by Vishnu-Lakkimsetty-E3640** [cloud-custodian/cloud-custodian#9383] [AWS]
Hi Team,

I'm looking for possibilities to pass the required lambda memory and ephemeral storage values within a custodian or a configuration. Could someone please help me with this?

Regards,
Vishnu L

**Comment by ajkerrigan** [comment]
Cloud Custodian's [Lambda Configuration](https://cloudcustodian.io/docs/aws/lambda.html#lambda-configuration) page has some information about this, and you can find the full schema for any specific policy execution mode on the [AWS Execution Modes](https://cloudcustodian.io/docs/aws/resources/aws-modes.html#aws-modes) page.

In your case, you can customize memory by providing a value for `memory` in your `mode` block:

```yaml
mode:
  type: cloudtrail
  events:
    - ...
  role: ...
  memory: 1024
```

At the moment it doesn't look like we support overriding ephemeral storage, so that would need a code change.
